internal/cli/atlas/quickstart/confirm_cluster_setup.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package quickstart
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+func (opts *Opts) askConfirmConfigQuestion() error {
+	if opts.Confirm {
+		return nil
+	}
+
+	diskSize := uint64(0)
+	if opts.newCluster().DiskSizeGB != nil {
+		diskSize = uint64(*opts.newCluster().DiskSizeGB)
+	}
+
+	fmt.Printf(`
+[Summary of changes]
+Name:			%s
+Tier:			%s
+Disk Size (GB):		%d
+User:			%s
+Ip Address:		%s
+`,
```

Comment:
```suggestion
Cluster Name:			%s
Cluster Tier:			%s
Cluster Disk Size (GB):		%d
Database Username:			%s
Allowed connections from (IP Address):		%s
`,
```

And can we please add Cloud provider and region?

Reply:
Sure, anything else we might need? (above was what was in the ticket, not sure what's important to our users)

Reply:
also super nit `opts.IPAddresses` is a slice and why it adds brackets on your example, wondering if we should format it better, `strings.Join` kind of thing

Reply:
I actually quite like the output with the brackets (it felt like the right representation to me): 
```
User:			Quickstart-1632155644
Ip Address:		[109.255.179.192]
```

Reply:
> anything else we might need? 

main feedback was tier and disk size as it's "hidden" but based on personal experience with a different tool all things a user can pick should be shown to give them/us a change to double check those are the right settings

so maybe a yes/no to the `load sample data` and open shell questions?

Reply:
Settled on the `strings.join` format without the brackets